file_formats/cmo_format.mli

```diff
@@ -53,6 +53,7 @@ type compilation_unit =
 type library =
   { lib_units: compilation_unit list;   (* List of compilation units *)
     lib_custom: bool;                   (* Requires custom mode linking? *)
+    (* The data in the following fields is stored in *reverse* order. *)
```

Comment:
The comment is not explicit about what order it is the reverse of. Your are implicitly assuming that the natural order is the command-line parameter order (plus, I guess, eventual environment cascades etc.), and it would be nice if there was a (concise) way to make this order explicit.

Reply:
Ok tried something else.